RoadTripMoustache/kana_to_kanji
# feat: Kana, kanji and vocabulary details skeleton

## 📖 Description


Implement the skeleton for kana, kanji, and vocabulary details view. 
Please note that TTS will be added in a second PR 😄 

This will be loaded as a modal bottom sheet adjusted based on the content.

### ⁉️ Related Issues




- linked to #102 


### 🖼️ Screenshots:


<details>
    <summary>Demo!</summary> 

https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/e30e6852-de5e-422e-ad0d-714df98f9763

</details>

### 🧪 How to test the change?




1. Go to _Glossary_ tab
2. Select any kana, kanji, or vocabulary
3. See changes

### ☑️ Checklist before requesting a review
